src/compiler/checker.ts

```diff
@@ -38301,7 +38301,9 @@ namespace ts {
         }
 
         function checkExportAssignment(node: ExportAssignment) {
-            if (checkGrammarModuleElementContext(node, Diagnostics.An_export_assignment_can_only_be_used_in_a_module)) {
+            if (checkGrammarModuleElementContext(node, node.isExportEquals

```

Comment:
This is weird because `node.isExportEquals` looks like the second argument at a glance.